src/main/scala/com/fulcrumgenomics/bam/ClipBam.scala

```diff
@@ -30,20 +30,23 @@ import com.fulcrumgenomics.bam.api.{SamOrder, SamRecord, SamSource, SamWriter}
 import com.fulcrumgenomics.cmdline.{ClpGroups, FgBioTool}
 import com.fulcrumgenomics.commons.util.LazyLogging
 import com.fulcrumgenomics.sopt.{arg, clp}
-import com.fulcrumgenomics.util.{Io, ProgressLogger}
+import com.fulcrumgenomics.util.{Io, Metric, ProgressLogger}
+import enumeratum.EnumEntry
 import htsjdk.samtools.SAMFileHeader.SortOrder
+import htsjdk.samtools.SamPairUtil
 import htsjdk.samtools.SamPairUtil.PairOrientation
-import htsjdk.samtools._
 import htsjdk.samtools.reference.ReferenceSequenceFileWalker
 
+import scala.collection.immutable.IndexedSeq
+
 @clp(group = ClpGroups.SamOrBam, description=
   """
     |Clips reads from the same template. Ensures that at least N bases are clipped from any end of the read (i.e.
     |R1 5' end, R1 3' end, R2 5' end, and R2 3' end).  Optionally clips reads from the same template to eliminate overlap
     |between the reads.  This ensures that downstream processes, particularly variant calling, cannot double-count
     |evidence from the same template when both reads span a variant site in the same template.
     |
-    |Clipping overlapping reads is only performed on `FR` read pairs, and is implemented by clipping approximately half
+    |Clipping overlapping reads is only performed on `FR` reads, and is implemented by clipping approximately half
```

Comment:
I think this should still say "read pairs", right?  since `FR` reads doesn't quite make sense.

Reply:
Search and replace error.